libs/package.json

```diff
@@ -20,8 +20,8 @@
   "dependencies": {
     "@audius/hedgehog": "1.0.8",
     "@ethersproject/solidity": "5.0.5",
-    "@solana/spl-token": "^0.1.4",
-    "@solana/web3.js": "1.18.0",
+    "@solana/spl-token": "0.1.6",
+    "@solana/web3.js": "1.20.0",
```

Comment:
[current master version is at 1.18.0](https://github.com/AudiusProject/audius-protocol/blob/master/libs/package-lock.json#L689), but that breaks CN tests so i've bumped it up to latest version 1.20.0, which makes all tests pass